tests/test_utils.py

```diff
@@ -159,22 +161,55 @@
     "bio_things": {"class": "BioThings", "property": "bioThings"},
 }
 
+test_disk_storage = [
+    (2, 4000, 16000),
+    (1000, 4000, 16000),
+    (2000000, 1900000, 2000000),
+    (1073741825, 1073741824, 1181116006.4),
+]
+
+
+# create temporary files with various size based on request
+@pytest.fixture()
+def create_temp_query_file(
```

Comment:
@andrewelamb could you double check the type hinting here? Does this look correct? i have never added type hinting to fixture so I am not so sure.